seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/TaskExecutionService.java

```diff
@@ -180,6 +182,16 @@ public PassiveCompletableFuture<TaskExecutionState> deployLocalTask(
             logger.severe(ExceptionUtils.getMessage(t));
             resultFuture.completeExceptionally(t);
         }
+        resultFuture.whenComplete((r, s) -> {
+            InvocationFuture<Object> invoke;
+            do {
```

Comment:
I think we should add a sleep time before retry.

Reply:
Thanks. I fixed